addons/rating/static/tests/qunit_suite_tests/components/thread_needaction_preview_tests.js

```diff
@@ -0,0 +1,60 @@
+/** @odoo-module **/
+
+import { afterNextRender, start, startServer } from '@mail/../tests/helpers/test_utils';
+
+QUnit.module('rating', {}, function () {
+QUnit.module('components', {}, function () {
+QUnit.module('thread_needaction_preview_tests.js');
+
+QUnit.test('rating value displayed on the thread needaction preview', async function (assert) {
+    assert.expect(3);
+    const pyEnv = await startServer();
+    const resPartnerId1 = pyEnv['res.partner'].create();
+    const mailMessageId1 = pyEnv['mail.message'].create({
+        model: 'res.partner', // Can be 'project.task' in actual operation.
```

Comment:
should maybe move test to `test_mail(_full)` and use one of the test models with rating instead?

Reply:
but this is not really testing the full rating part, such as the rating in the message or something like that. This is more focused on the thread_needaction_preview or so. if move to test_mail_all, then should get more test on the whole rating, which should be another pr later for that?( such as when introducing rating model)